internal/storage/metricstore/elasticsearch/README.md

```diff
@@ -0,0 +1,223 @@
+## `getCallRate` Calculation Explained
+
+The `getCallRate` method calculates the call rate (requests per second) for a service by querying span data stored in Elasticsearch. The process involves three key stages: filtering the relevant spans, performing a time-series aggregation to count requests, and post-processing the aggregated data to calculate the final rate.
+
+This document breaks down each of these stages, referencing the corresponding parts of the Elasticsearch query and the Go implementation.
+
+### 1\. Filter Query Part
+
+The first step is to isolate the specific set of documents (spans) needed for the calculation. We use a `bool` query with a `filter` clause, which is efficient as it doesn't contribute to document scoring.
+
+**ES Query Reference:**
+
+```json
+"query": {
+  "bool": {
+    "filter": [
+      { "term": { "process.serviceName": "${service}" } },
+      {
+        "nested": {
+          "path": "tags",
```

Comment:
Would you like to look into the other issue first about materializing first-class OTLP span fields into top-level ES doc fields? 

Without that your implementation will not work if the users configure their Jaeger with `allTagsAsFields` or include the `span.kind` tag in `tagKeysAsFields` list. You can see in `spanstore/writer.go/splitElevatedTags` function that it clearly partitions the original tags into a nested table and a map.